### ISSUE TITLE
Missing trigger in sample

### ISSUE THREAD

**Issue by philippegabert** [TIBCOSoftware/flogo-cli#73]
Hi, 

importing the following app:
https://github.com/TIBCOSoftware/flogo-cli/blob/master/docs/samples/devices/device-l0x.json

gives me an error because the trigger "has not been installed"

Actually, it seems that it is just because the path to the trigger is not correct.

Instead of:
`{
      "id": "l0x_trigger",
      "name": "Read From VL53L0X",
      "ref": "github.com/TIBCOSoftware/flogo-contrib/device/trigger/vl53l0x_stream",
      "actionId": "monitor_dist",
      "settings": {
        "interval": "500"
      }
    }`

it should be:
`{
      "id": "l0x_trigger",
      "name": "Read From VL53L0X",
      "ref": "github.com/TIBCOSoftware/flogo-contrib/device/trigger/vl53l0xstream",
      "actionId": "monitor_dist",
      "settings": {
        "interval": "500"
      }
    }`

Philippe

**Comment by mellistibco** [comment]
@philippegabert actually it looks like that ref path is fine, but the problem is in the flogo-contrib with the trigger.json (https://github.com/TIBCOSoftware/flogo-contrib/blob/master/device/trigger/vl53l0x_stream/trigger.json). The ref in the json metadata is incorrect.

It should be:

```json
"ref": "github.com/TIBCOSoftware/flogo-contrib/device/trigger/vl53l0x_stream",
```

**Comment by mellistibco** [comment]
resolved in TIBCOSoftware/flogo-contrib#225
